plugins/synaptics-rmi/fu-synaptics-rmi-hid-device.c

```diff
@@ -392,30 +393,36 @@ fu_synaptics_rmi_hid_device_rebind_driver(FuSynapticsRmiDevice *self, GError **e
 		return FALSE;
 	}
 
+	/* build paths */
```

Comment:
i think you need to rebase this against main and `git push --force` to update this PR :)